examples/bpmn-utlis.js

```diff
@@ -0,0 +1,577 @@
+                <di:waypoint x="192" y="99" />
+                <di:waypoint x="250" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNEdge id="Flow_0wu2yg6_di" bpmnElement="Flow_0wu2yg6">
+                <di:waypoint x="350" y="99" />
+                <di:waypoint x="410" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNEdge id="Flow_1gcjvke_di" bpmnElement="Flow_1gcjvke">
+                <di:waypoint x="510" y="99" />
+                <di:waypoint x="570" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNEdge id="Flow_1bmck4z_di" bpmnElement="Flow_1bmck4z">
+                <di:waypoint x="670" y="99" />
+                <di:waypoint x="730" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNEdge id="Flow_1i7r7y7_di" bpmnElement="Flow_1i7r7y7">
+                <di:waypoint x="830" y="99" />
+                <di:waypoint x="890" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNEdge id="Flow_12yysoe_di" bpmnElement="Flow_12yysoe">
+                <di:waypoint x="990" y="99" />
+                <di:waypoint x="1062" y="99" />
+              </bpmndi:BPMNEdge>
+              <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_07yal9c">
+                <dc:Bounds x="156" y="81" width="36" height="36" />
+                <bpmndi:BPMNLabel>
+                  <dc:Bounds x="162" y="124" width="24" height="14" />
+                </bpmndi:BPMNLabel>
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Activity_1xr04un_di" bpmnElement="Activity_1xr04un">
+                <dc:Bounds x="250" y="59" width="100" height="80" />
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Activity_03oln2l_di" bpmnElement="Activity_0kn4d46">
+                <dc:Bounds x="410" y="59" width="100" height="80" />
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Activity_0owvwg0_di" bpmnElement="Activity_1j15wcw">
+                <dc:Bounds x="570" y="59" width="100" height="80" />
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Activity_10z8211_di" bpmnElement="Activity_0wbisol">
+                <dc:Bounds x="730" y="59" width="100" height="80" />
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Activity_1aadlt5_di" bpmnElement="Activity_0y3sd80">
+                <dc:Bounds x="890" y="59" width="100" height="80" />
+              </bpmndi:BPMNShape>
+              <bpmndi:BPMNShape id="Event_157byoi_di" bpmnElement="Event_157byoi">
+                <dc:Bounds x="1062" y="81" width="36" height="36" />
+                <bpmndi:BPMNLabel>
+                  <dc:Bounds x="1070" y="124" width="20" height="14" />
+                </bpmndi:BPMNLabel>
+              </bpmndi:BPMNShape>
+            </bpmndi:BPMNPlane>
+          </bpmndi:BPMNDiagram>
+        </bpmn:definitions>
+        `;
+}
+
+function getVerticalBpmnDiagram() {
```

Comment:
:question: I suggest we state that it is a custom diagram to show fit capabilities

Reply:
Done